### PR TITLE
1.115.0

### PR DESCRIPTION
**Pull request by mbtools**
Preparation for v1.115.0

### CODE REVIEW

**Comment by larshp** [comment]
just go for it 👍 

update https://github.com/abapGit/abapGit/blob/main/src/zif_abapgit_version.intf.abap and then we can merge

**Comment by mbtools** [comment]
@larshp ready to go. just some ajson UT is failing in npm

File: changelog.txt [hunk 1]
```diff
@@ -8,6 +8,91 @@ Legend
...
+* SOTS: Fix dump in case of lower case language code (#5381)
+* Fix DDIC errors related to ref to class/interface (#5369)
+* TTYP: Fix pull/activate table type ref to data type (#5384)
+* CLAS/INTF: Fix serialize/deserialize of documentation (#5374)
+* WEBI: Sort tables (#5390)
+* SRVD: Update zcl_abapgit_object_srvd.clas.abap (#5395)
+* SRVB: Fix uncaught exception (#5398)
+* Repo List: Fix sort (#5397)
+* Fix deserialize order (#5399)
+* ajson, Automatic Update (#5412)
```
Comment (larshp) [inline changelog.txt:67]: not a user facing or programming change

```suggestion
```

File: changelog.txt [hunk 2]
```diff
@@ -8,6 +8,91 @@ Legend
...
+* Improve LXE error handling (#5440)
+* UI: Fix escaping of textarea input (#5456)
+* Improve JSON syntax highlighter (#5450)
+* DEVC: Fix DYNPRO_SEND_IN_BACKGROUND (#5461)
+* Avoid logon when comparing with remote (#5458)
+* DDLS: Clear ABAP language version (#5474)
+* DDLX: Clear ABAP language version (#5476)
+* FUGR: Fix diff of includes (#5489)
+* DTDC: Support ABAP 2208 (#5478)
+* Create empty classes/interfaces before DDIC (#5393)
```
Comment (larshp) [inline changelog.txt:90]: this could be a breaking change?

```suggestion
! Create empty classes/interfaces before DDIC (#5393)
```

Comment (mbtools) [reply changelog.txt:89]: didn't break anything in CI but might somewhere else

**Review by larshp — APPROVED** [review]

**Comment by abap-weasel** [comment]
Thanks guys! 👍

**Pull request merged**